src/larecs/world.mojo

```diff
@@ -1040,12 +980,12 @@ struct World[*component_types: ComponentType](Copyable, Movable, Sized):
         ```
 
         Parameters:
-            has_without_mask: Whether the query has a without mask.
+            has_exclude: Whether the filter has an exclude mask.
             Ts: The types of the components to add.
 
         Args:
-            query: The query specifying which entities to modify. The query must explicitly exclude existing entities
-                that already have some of the components to add.
+            filter: The [..filter.MaskFilter] specifying which entities to modify. The filter must explicitly exclude
+                existing entities that already have some of the components to add.
```

Comment:
Is the "must explicitly exclude" still true?

Also, maybe mention the option to use a query here.